src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -22137,6 +22137,7 @@ a[dir]:root[data-theme=default] .badge-dark:focus, a[data-theme=default][dir]:ro
 :root[data-theme=default] .ta-navbar-top a {
   color: #495057;
   text-decoration: none;
+  cursor: pointer;
```

Comment:
Why are these CSS changes necessary?

Reply:
Ok, we need to change this because I'm getting admin UI errors.

First thing. Don't make the changes directly in the .css files. It needs to be added in the SCSS files and then transpiled so that it is also included in the minified version. I will create a Pull Request to fix this and remove the changes dones directly in this file by re-transpiling the SCSS assets.

Reply:
Argh, I didn't notice that in this project we have SCSS...